.github/workflows/contributing_update_all.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Update all Contributing Files"
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Every Sunday at midnight UTC
+  workflow_dispatch:
+
+jobs:
+  update-contributing:
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/contributing_update_all.yaml@main
+    with:
+      branch: 'main'
```

Comment:
Let's add a new line at the end of the file, please.

Reply:
Done.